src/App.ts

```diff
@@ -969,9 +987,18 @@ export default class App<AppCustomContext extends StringIndexed = StringIndexed>
       }
     }
 
+    /** 
+     * Set the Bot Access Token if it exists in event payload
+     * Sometimes the bot_access_token is located in the event
+     * Sometimes it is located directly in the body.
+     */
+    let botAccessToken = body.bot_access_token ?? body.event.bot_access_token;
```

Comment:
This PR introduces the existence of bot_access_token and assigns it to the event context below. This is passed in function interactivity payloads (today that includes `function_execution` `block_actions`, `view_*`) events. 

When this token is provided, we want to override the client token to use this over regular xoxb.